### PR TITLE
Do not allow canvas size to become smaller than MinSize in wx backend…

### PR DESCRIPTION
**Pull request by jmoraleda**
Bug fix for issue #14883 as discussed there.

### CODE REVIEW

**Comment by tacaswell** [comment]
The travis failures are all failure to install ffmpeg which I hope are transient.

Thanks @jmoraleda !

File: lib/matplotlib/backends/backend_wx.py [hunk 1]
```diff
@@ -768,6 +768,7 @@ def _onSize(self, evt):
         else:
             # variable size
             size = self.GetClientSize()
+            size.IncTo(self.GetMinSize()) # Do not allow size to become smaller than MinSize
```
Comment (QuLogic) [inline lib/matplotlib/backends/backend_wx.py]: This won't pass flake8 once Travis is working again; two spaces before `#` and the line is too long.

**Comment by tacaswell** [comment]
Someone restarted travis at it is just as @QuLogic said, there is a flake8 failure.  Could you please fix that @jmoraleda ?

**Comment by jmoraleda** [comment]
I jjust fixed the problem. My personal pylint defaults were different than those used at the project, which made my comment not conform with the project convention, and it escaped me the first time around. Thank you.

**Pull request merged**